tests/SideBySide.New/ConnectionPool.cs

```diff
@@ -80,37 +80,7 @@ public void ResetConnection(object connectionReset, int poolSize, long expected)
 		}
 
 		[Fact]
-		public async Task ExhaustConnectionPoolWithTimeout()
```

Comment:
I believe this test to be incorrect.  I think that ConnectionTimeout is a timeout for a TCP Connection to be made to the MySQL server.  I don't think it applies to a pooled connection being retrieved from the pool.